service/src/main/java/bio/terra/landingzone/stairway/flight/create/CreateLandingZoneFlight.java

```diff
@@ -41,10 +51,11 @@ private void addCreateSteps(LandingZoneFlightBeanBag flightBeanBag) {
             flightBeanBag.getAzureLandingZoneManagerProvider(), flightBeanBag.getObjectMapper()),
         RetryRules.shortExponential());
 
-    addStep(
-        new CreateAzureLandingZoneStep(flightBeanBag.getAzureLandingZoneManagerProvider()),
-        RetryRules.cloud());
-
+    if (!requestedLandingZone.isAttaching()) {
+      addStep(
+          new CreateAzureLandingZoneStep(flightBeanBag.getAzureLandingZoneManagerProvider()),
+          RetryRules.cloud());
+    }
     addStep(
         new ResetResourceGroupTagsStep(
```

Comment:
Out of scope of this PR, but wondering if we still need the read/reset tags steps now that we're not relying on the RG tag to store the billingProfileId.

Reply:
Yup, will be removed shortly https://broadworkbench.atlassian.net/browse/WOR-810